### ISSUE TITLE
Failed to compile - ERROR TS2305

### ISSUE THREAD

**Issue by ShaneCotter** [aws-amplify/amplify-js#2294]
ERROR TS2305 

When I run my project it produces the following error : 

> [ng] ERROR in node_modules/aws-amplify-angular/lib/components/storage/photo-picker-component/photo-picker.factory.d.ts(3,10): error TS2305: Module '"C:/Users/Shane/aws-amplify-ionic-sample/node_modules/aws-sdk/clients/cognitoidentity"' has no exported member 'String'.

I'm aware the previously report fix for this issue was to change

`import { String } from 'aws-sdk/clients/cognitoidentity';
`

to 

`import * as String from 'aws-sdk/clients/cognitoidentity';`

But there is no String import in my photo-picker.factory.d.ts file.

Here is what is in my photo-picker.factory.d.ts file:

```
import { OnInit, ComponentFactoryResolver, OnDestroy, EventEmitter } from '@angular/core';
import { DynamicComponentDirective } from '../../../directives/dynamic.component.directive';
export declare class PhotoPickerComponent implements OnInit, OnDestroy {
    private componentFactoryResolver;
    framework: string;
    url: string;
    path: string;
    storageOptions: any;
    picked: EventEmitter<string>;
    loaded: EventEmitter<string>;
    uploaded: EventEmitter<Object>;
    componentHost: DynamicComponentDirective;
    constructor(componentFactoryResolver: ComponentFactoryResolver);
    ngOnInit(): void;
    ngOnDestroy(): void;
    loadComponent(): void;
}
```

Can someone please help ?

**Comment by ShaneCotter** [comment]
Issue #1442 previously resolved this issue but the solution to #1442 does not resolve my issue.

**Comment by haverchuck** [comment]
@ShaneCotter  - Can you please post the versions of aws-amplify and aws-amplify-angular that you are using?  Also - I'm assuming by the path in your error that this is an Ionic project?
